packages/block-library/src/template-part/edit/import-controls.js

```diff
@@ -145,33 +146,35 @@ export function TemplatePartImportControls( { area, setAttributes } ) {
 	}
 
 	return (
-		<HStack as="form" onSubmit={ createFromWidgets }>
-			<FlexBlock>
-				<SelectControl
-					label={ __( 'Import widget area' ) }
-					value={ selectedSidebar }
-					options={ options }
-					onChange={ ( value ) => setSelectedSidebar( value ) }
-					disabled={ ! options.length }
-					__next36pxDefaultSize
-					__nextHasNoMarginBottom
-				/>
-			</FlexBlock>
-			<FlexItem
-				style={ {
-					marginBottom: '8px',
-					marginTop: 'auto',
-				} }
-			>
-				<Button
-					variant="primary"
-					type="submit"
-					isBusy={ isBusy }
-					aria-disabled={ isBusy || ! selectedSidebar }
+		<Spacer marginBottom="4">
```

Comment:
I know this isn't the best way to solve a spacing problem, but another alternative is to change the Block Inspector CSS